app/src/main/java/com/launcher/silverfish/launcher/appdrawer/TabbedAppDrawerFragment.java

```diff
@@ -313,15 +300,22 @@ public boolean onDrag(View view, DragEvent dragEvent) {
                     }
 
                     case DragEvent.ACTION_DROP: {
+
+                        dragOffsetX = dragBeginX-dragEvent.getX();  // Total x movement
```

Comment:
Should add a space around the - operator to be consistent with the coding style adopted in your Utils.java changes.